config/schemas/config.yaml

```diff
@@ -1363,6 +1372,727 @@ properties:
+      letsencrypt:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          prod:
+            additionalProperties: false
+            properties:
+              email:
+                type: string
+              solvers:
+                type: array
+            type: object
+          staging:
+            additionalProperties: false
+            properties:
+              email:
+                type: string
+              solvers:
+                type: array
+            type: object
+        type: object
+    type: object
+  networkPolicies:
+    additionalProperties: false
+    properties:
+      additional:
+        type: object
+      additionalEgressPolicies:
+        type: array
+      additionalIngressPolicies:
+        type: array
+      alertmanager:
+        additionalProperties: false
+        properties:
+          alertReceivers:
+            $ref: '#/$defs/netpolRule'
+          enabled:
+            type: boolean
+        type: object
+      allowedNameSpaces:
+        type: array
+      certManager:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          letsencrypt:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+          namespaces:
+            items:
+              default: dex
+              type: string
+            type: array
+          dns01:
+            type: object
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+        type: object
+      coredns:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+          externalDns:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+          serviceIp:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+            type: object
+        type: object
+      defaultDeny:
+        default: false
+        type: boolean
+      dex:
+        additionalProperties: false
+        properties:
+          connectors:
+            additionalProperties: false
+            properties:
+              ips:
+                $ref: '#/$defs/iplist'
+              ports:
+                $ref: '#/$defs/portlist'
+                default:
+                  - 443
+            type: object
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      dnsAutoscaler:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: true
+            type: boolean
+        type: object
+      enableAlerting:
+        default: true
+        type: boolean
```

Comment:
We should probably consider refactoring this. So that `enableAlerting` becomes `alerting.enabled`. But that is different task then this one.